actions/src/pull_requests/ciChecks.js

```diff
@@ -38,21 +42,18 @@ const handleFailure = async (context) => {
       // pull request data, hence we need to fetch the pull request
       // so we can get the author.
       const pullRequestData = checkSuite.pull_requests[0];
-      const pullRequestResponse = await context.github.pulls.get(
-        context.repo({
+      const pullRequestResponse = await octokit.pulls.get(
+        {
           pull_number: pullRequestData.number,
-        })
+          ...context.repo
+        }
       );
       const pullRequest = pullRequestResponse.data;
       const prAuthor = pullRequest.user.login;
 
       const commentBody =
-        'Hi @' +
-        prAuthor +
-        ', there are some failing CI checks in your latest push ' +
-        ' If you think this is due to a flake, please file an issue ' +
-        'before restarting the tests. Thanks!';
-      await pingAndAssignUsers(context, pullRequest, [prAuthor], commentBody);
+        'Hi @' + prAuthor + ', there are some failing CI checks in your latest push ' + ' If you think this is due to a flake, please file an issue ' + 'before restarting the tests. Thanks!';
```

Comment:
Make sure to limit line lengths under 80 chars.

Reply:
done